cmd/influxd/launcher/launcher.go

```diff
@@ -269,6 +269,18 @@ func buildLauncherCommand(l *Launcher, cmd *cobra.Command) {
 			Default: "",
 			Desc:    "TLS key for HTTPs",
 		},
+		{
+			DestP:   &l.httpTLSMinVersion,
+			Flag:    "tls-min-version",
+			Default: "1.2",
+			Desc:    "Minimum accepted TLS version",
+		},
+		{
+			DestP:   &l.httpTLSStrictCiphers,
+			Flag:    "tls-strict-ciphers",
+			Default: false,
+			Desc:    "Restrict accept ciphers to: ECDHE_RSA_WITH_AES_256_GCM_SHA384, ECDHE_RSA_WITH_AES_256_CBC_SHA, RSA_WITH_AES_256_GCM_SHA384, RSA_WITH_AES_256_CBC_SHA",
```

Comment:
One thing we did to avoid the boolean behaviour being changed over time: Allow the user to define the cipher list as a string, and set a strong default ourselves. AFAIK this is the same in Apache/Nginx.

This may need additional cipher verification but also allows more fine grained control for enterprise environments.

Reply:
I did consider that, but was keen for InfluxDB to provide opinionated and strong defaults; that hopefully change with the times.

Reply:
Makes sense. And you possibly don't have to deal with strange openssl C api bugs on ciphers (not) loaded on rhel7, as I had fun with in icinga2. That bug caused all our default secure ciphers to be useless and no handshakes performed then.